test/extended/builds/build_metrics.go

```diff
@@ -0,0 +1,57 @@
+package builds
+
+import (
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	exutil "github.com/openshift/openshift-tests/test/extended/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[sig-apps] Workloads", func() {
```

Comment:
@wewang58 could you please move the case to openshift-test-private repo? there are some workloads cases in there. cc @zhouying7780

Reply:
@kuiwang02  Thanks reviewing it, will remove to openshift-test-private repo.